.version

```diff
@@ -0,0 +1 @@
+5.0.0
```

Comment:
This file is the only source of version information. I could use the CI build scripts, but it will not work from local builds.